packages/nextjs/components/scaffold-eth/Contract/utilsContract.tsx

```diff
@@ -2,9 +2,9 @@ import { AbiFunction, AbiParameter } from "abitype";
 import { BaseError as BaseViemError } from "viem";
 
 /**
- * @dev utility function to generate key corresponding to function metaData
+ * Generates a key based on function metadata
  * @param {AbiFunction} functionName
```

Comment:
Thanks 🙌 updated at [d1cbac1](https://github.com/scaffold-eth/scaffold-eth-2/pull/627/commits/d1cbac11b2bf0e596df9c5847a102763f6789fec)

I think maybe in other PR we can remove all JsDoc types comments since we are already using typescript and it kind of serves the same purpose,  will add it in #617